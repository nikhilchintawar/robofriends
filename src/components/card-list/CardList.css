.cardList{
    display: grid;
    grid-template-columns: repeat(4, 1fr);
    row-gap: 25px;
    column-gap: 10px;
    margin: 220px 10px 0px 10px;
}

@media screen and (max-width: 600px){
    .cardList{
        display: grid;
        grid-template-columns: repeat(1, 1fr);
        
    }
}