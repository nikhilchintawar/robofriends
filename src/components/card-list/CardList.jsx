import React from "react";
import Card from '../card/Card';
import './CardList.css';

const CardList = ({ robots })=> {
//   if (true) {
//       throw new Error("hahaha")
//   }
    return (
        <div className="cardList">
        {
        robots.map((users, i) => {
        return (<Card 
        key={i} 
        id={robots[i].id} 
        name={robots[i].name} 
        email={robots[i].email} />)
    })
        }
        </div>
    )
}


export default CardList;