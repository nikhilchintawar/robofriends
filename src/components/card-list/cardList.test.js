import { shallow } from 'enzyme';
import React from 'react';

import CardList from "./CardList";

const filteredRobots = [{
    id: 1,
    name: 'Leanne Graham',
    username: 'Bret',
    email: 'Sincere@april.biz'
  }]

test('test to render card list component.', () => {
    expect(shallow(<CardList robots={filteredRobots} />)).toMatchSnapshot()
})