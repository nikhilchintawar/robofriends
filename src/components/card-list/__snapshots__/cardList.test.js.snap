// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to render card list component. 1`] = `ShallowWrapper {}`;
